packages/block-library/src/image/edit.js

```diff
@@ -108,11 +108,17 @@ export function ImageEdit( {
 		align,
 		metadata,
 	} = attributes;
+
 	const [ temporaryURL, setTemporaryURL ] = useState( attributes.blob );
 
 	const [ contentResizeListener, { width: containerWidth } ] =
 		useResizeObserver();
 
+	const [ placeholderResizeListener, { width: placeholderWidth } ] =
+		useResizeObserver();
+
+	const isSmallContainer = placeholderWidth ? placeholderWidth < 160 : false;
```

Comment:
```suggestion
	const isSmallContainer = placeholderWidth && placeholderWidth < 160;
```

Nit: I think this can be simplified.